nilearn/load_confounds/parser.py

```diff
@@ -0,0 +1,387 @@
+
+    motion : string, optional
+        Type of confounds extracted from head motion estimates.
+        "basic" translation/rotation (6 parameters)
+        "power2" translation/rotation + quadratic terms (12 parameters)
+        "derivatives" translation/rotation + derivatives (12 parameters)
+        "full" translation/rotation + derivatives + quadratic terms + power2d
+               derivatives (24 parameters)
+
+    n_motion : float
+        Number of pca components to keep from head motion estimates.
+        If the parameters is strictly comprised between 0 and 1, a principal
+        component analysis is applied to the motion parameters, and the number
+        of extracted components is set to exceed `n_motion` percent of the
+        parameters variance.
+        If the n_components = 0, then no PCA is performed.
+
+    fd_thresh : float, optional
+        Framewise displacement threshold for scrub (default = 0.2 mm)
+
+    std_dvars_thresh : float, optional
+        Standardized DVARS threshold for scrub (default = 3)
+
+    wm_csf : string, optional
+        Type of confounds extracted from masks of white matter and
+        cerebrospinal fluids.
+        "basic" the averages in each mask (2 parameters)
+        "power2" averages and quadratic terms (4 parameters)
+        "derivatives" averages and derivatives (4 parameters)
+        "full" averages + derivatives + quadratic terms
+               + power2d derivatives (8 parameters)
+
+    global_signal : string, optional
+        Type of confounds extracted from the global signal.
+        "basic" just the global signal (1 parameter)
+        "power2" global signal and quadratic term (2 parameters)
+        "derivatives" global signal and derivative (2 parameters)
+        "full" global signal + derivatives + quadratic terms
+               + power2d derivatives (4 parameters)
+
+    scrub : string, optional
+        Type of scrub of frames with excessive motion (Power et al. 2014)
+        "basic" remove time frames based on excessive FD and DVARS
+        "full" also remove time windows which are too short after scrubbing.
+        one-hot encoding vectors are added as regressors for each scrubbed
+        frame.
+
+    compcor : string, optional
+        Require fmriprep >= v:1.4.0.
+        Type of confounds extracted from a component based noise correction
+        method
+        "anat" noise components calculated using anatomical compcor
+        "temp" noise components calculated using temporal compcor
+        "full" noise components calculated using both temporal and anatomical
+
+    n_compcor : int or "auto", optional
+        The number of noise components to be extracted.
+        For acompcor_combined=False, and/or compcor="full", this is the number
+        of components per mask.
+        Default is "auto": select all components
+        (50% variance explained by fMRIPrep defaults)
+
+    acompcor_combined: boolean, optional
```

Comment:
The name is not great. Couldn't it be an option in `compcor` above ?

Reply:
In that case we will have five options:

1. `temp`
2. `anat_separated` (WM compcor and CSF compcor concatenated)
3. `anat_combined` (use compcor calculated on a WM+CSF mask)
4. `temp_anat_separated` (`compcor='full', acompcor_combined=False`)
5. `temp_anat_combined`(`compcor='full', acompcor_combined=True`)

Better idea for the option names?

Reply:
yes, replacing `temp` with  `temporal`